### PR TITLE
docs: Add wasm manifest content to splash screen feature docs

### PR DESCRIPTION
**Pull request by lukeblevins**
GitHub Issue (If applicable): contributes toward #14805 

## PR Type

What kind of change does this PR introduce?

- Documentation content changes

## What is the current behavior?

Bootstrapper-oriented [doc](https://github.com/unoplatform/Uno.Wasm.Bootstrap/blob/8a7d79d426e61d282d28f4966ac4ec2298a9ff95/doc/features-splash-screen.md) overlap with existing feature content.

## What is the new behavior?

Improved the general feature doc to contain content based on a [topic](https://platform.uno/docs/articles/external/uno.wasm.bootstrap/doc/features-splash-screen.html) in the wasm bootstrapper repo

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

@jeromelaban Assuming these changes are acceptable, would it make sense to remove/repurpose the existing [article](https://github.com/unoplatform/Uno.Wasm.Bootstrap/blob/8a7d79d426e61d282d28f4966ac4ec2298a9ff95/doc/features-splash-screen.md) last edited 2 years ago?

Internal Issue (If applicable): N/A

### CODE REVIEW

File: doc/articles/splash-screen.md [hunk 1]
```diff
@@ -150,46 +150,48 @@ While the new templates simplify adding a splash screen, this article covers how
 
 * The default splash screen configuration for WebAssembly is to use the Uno Platform logo as a placeholder
 
-* In the `.Wasm` project, navigate to `WasmScripts/AppManifest.js`
+* An `AppManifest.js` file contains settings for your WebAssembly application, including properties to customize its splash screen. This file found in the `[AppName].Wasm` project typically located at `WasmScripts/AppManifest.js`
```
Comment (Youssef1313) [inline doc/articles/splash-screen.md:153]:
```suggestion
* An `AppManifest.js` file contains settings for your WebAssembly application, including properties to customize its splash screen. This file is found in the `[AppName].Wasm` project, typically located at `WasmScripts/AppManifest.js`.
```

File: doc/articles/splash-screen.md [hunk 2]
```diff
@@ -150,46 +150,48 @@ While the new templates simplify adding a splash screen, this article covers how
...
 
-  * Customize the splash screen image and background color by setting the following properties related to splash screens
+    #### General properties
```
Comment (Youssef1313) [inline doc/articles/splash-screen.md:155]:
```suggestion
#### General properties
```

File: doc/articles/splash-screen.md [hunk 3]
```diff
@@ -150,46 +150,48 @@ While the new templates simplify adding a splash screen, this article covers how
...
 
-#### Standard properties for splash screens
+    Customize the splash screen image and background color by adjusting several key properties:
```
Comment (Youssef1313) [inline doc/articles/splash-screen.md:157]:
```suggestion
You can customize the splash screen image and background color by adjusting several key properties:
```

File: doc/articles/splash-screen.md [hunk 4]
```diff
@@ -150,46 +150,48 @@ While the new templates simplify adding a splash screen, this article covers how
...
+    > `splashScreenColor` allows you to maintain a background color regardless of the system theme. However, a simple method to make the splash screen theme-aware is to assign `transparent` as its value or by omitting that property altogether.
 
-    ```js
-    var UnoAppManifest = {
-        splashScreenImage: "Assets/SplashScreen.scale-200.png",
-        splashScreenColor: "transparent",
-        displayName: "SplashScreenSample"
-    }
-    ```
+    #### Theme-aware properties
```
Comment (Youssef1313) [inline doc/articles/splash-screen.md:169]: All that should be dedented I think.

File: doc/articles/wasm-appmanifest.md [hunk 5]
```diff
@@ -4,31 +4,31 @@ uid: Uno.Development.WasmAppManifest
 
 # AppManifest for WebAssembly head project
 
-The `AppManifest.js` file contains settings for your WebAssembly application. It's normally located in the `[AppName].Wasm` project under the `WasmScripts` folder.
+The `[AppName].Wasm` project in your solution typically includes a manifest file containing settings for the WebAssembly application. This file is typically generated for you under `WasmScripts/AppManifest.js`.
 
 ## AppManifest properties
 
-The properties are :
+This app manifest file allows you to customize certain aspects of the WebAssembly application, including its loading screen. This loading UI is often referred to as the "splash screen" in documentation. See the WebAssembly [section](xref:Uno.Development.SplashScreen#5-webassembly) for more information, including how to customize using the properties in the app manifest.
```
Comment (Youssef1313) [inline doc/articles/wasm-appmanifest.md:11]:
```suggestion
This app manifest file allows you to customize certain aspects of the WebAssembly application, including its loading screen. This loading UI is often referred to as the "splash screen" in documentation. For more information, see [How to manually add a splash screen](xref:Uno.Development.SplashScreen#5-webassembly).
```

**Review by jeromelaban — APPROVED** [review]

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.1

**Pull request merged**